openmoc/map_to_dict.i

```diff
@@ -96,10 +96,10 @@
 }
 
 
-/* Typemap for Lattice::setUniverses(int num_x, int num_y, Universe** universes)
+/* Typemap for Lattice::setUniverses(int num_y, int num_x, Universe** universes)
```

Comment:
Ok, I'm a bit confused here. It looks intentional so you're probably correct, but at a first glance it looks like the old line was correct.